ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/vxlan-interface.j2

```diff
@@ -4,30 +4,54 @@
 
 ### VXLAN Interface Summary
 
-#### Source Interface: {{ vxlan_tunnel_interface.Vxlan1.source_interface }}
+#### Source Interface: {{ vxlan_tunnel_interface.Vxlan1.vxlan.source_interface }}
 
-#### UDP port: {{ vxlan_tunnel_interface.Vxlan1.vxlan_udp_port }}
+#### UDP port: {{ vxlan_tunnel_interface.Vxlan1.vxlan.udp_port }}
 
-#### VLAN to VNI Mappings
+{% if vxlan_tunnel_interface.Vxlan1.vxlan.virtual_router_encapsulation_mac_address is defined %}
 
-| VLAN | VNI |
-| ---- | --- |
-{%     for vlan in vxlan_tunnel_interface.Vxlan1.vxlan_vni_mappings.vlans | arista.avd.natural_sort %}
-| {{ vlan }} | {{ vxlan_tunnel_interface.Vxlan1.vxlan_vni_mappings.vlans[vlan].vni }} |
+#### EVPN MLAG Shared Router MAC : {{ vxlan_tunnel_interface.Vxlan1.vxlan.virtual_router_encapsulation_mac_address }}
+
+{% endif %}
+
+#### VLAN to VNI and Flood List Mappings
+
+| VLAN | VNI | Flood List |
+| ---- | --- | ---------- |
+{%     for vlan in vxlan_tunnel_interface.Vxlan1.vxlan.vlans | arista.avd.natural_sort %}
+| {{ vlan }} | {{ vxlan_tunnel_interface.Vxlan1.vxlan.vlans[vlan].vni }} | {{ vxlan_tunnel_interface.Vxlan1.vxlan.vlans[vlan].flood_vteps | join('<br/>') if vxlan_tunnel_interface.Vxlan1.vxlan.vlans[vlan].flood_vteps is defined else 'N/A' }}
```

Comment:
Maybe `arista.avd.default('N/A)` filter would be easier and shorter here.

Reply:
For this one, the simplest would be to use
```jinja
{{ vxlan_tunnel_interface.Vxlan1.vxlan.vlans[vlan].flood_vteps | arista.avd.default('N/A') }}
```
but this results in floodlist showing up as a list in documentation

```jinja
| VLAN | VNI | Flood List |
| ---- | --- | ---------- |
| 110 | 10110 | N/A |
| 111 | 10111 | N/A |
| 222 | 10222 | ['11.1.1.1', '11.1.1.2', '11.1.1.3'] |
```

if we add a line break tag, it affects strings also "N/A":

```jinja
{{ vxlan_tunnel_interface.Vxlan1.vxlan.vlans[vlan].flood_vteps | arista.avd.default('N/A') | join('<br/>') }}
```

```jinja
| VLAN | VNI | Flood List |
| ---- | --- | ---------- |
| 110 | 10110 | N<br/>/<br/>A |
| 111 | 10111 | N<br/>/<br/>A |
| 222 | 10222 | 11.1.1.1<br/>11.1.1.2<br/>11.1.1.3 |
```

Thus have gone with the following, which is a bit lengthy, but results in proper formatting:
```jinja
{{ vxlan_tunnel_interface.Vxlan1.vxlan.vlans[vlan].flood_vteps | join('<br/>') if vxlan_tunnel_interface.Vxlan1.vxlan.vlans[vlan].flood_vteps is arista.avd.defined else 'N/A' }} 
```

```jinja
| VLAN | VNI | Flood List |
| ---- | --- | ---------- |
| 110 | 10110 | N/A |
| 111 | 10111 | N/A |
| 222 | 10222 | 11.1.1.1<br/>11.1.1.2<br/>11.1.1.3 |
```

Reply:
Indeed, `default` filter generate one entry and `spit` does its job. So let's use your approach then.

Thanks for all the other changes.

Reply:
cool, sounds good.